ArdusatSDK.cpp

```diff
@@ -1278,14 +1523,15 @@ boolean UVLight::initialize(void) {
  * @brief   Takes a reading from the sensor
  * @ingroup uvlight
  */
-void UVLight::read(void) {
-  this->header.timestamp = millis();
-
+boolean UVLight::readSensor(void) {
   if (this->header.sensor_id == SENSORID_SI1132) {
```

Comment:
As with the other `readSensor` functions, use polymorphism to handle this, rather than checking our `sensor_id` value.